localflavor/br/models.py

```diff
@@ -18,3 +22,68 @@ def deconstruct(self):
         name, path, args, kwargs = super(BRStateField, self).deconstruct()
         del kwargs['choices']
         return name, path, args, kwargs
+
+
+class BRCPFField(CharField):
+    """
+    A model field for the brazilian document named of CPF (Cadastro de Pessoa Física)
+    """
+
+    description = _("CPF Document")
+
+    def __init__(self, *args, **kwargs):
+        kwargs['max_length'] = 14
+        super(BRCPFField, self).__init__(*args, **kwargs)
+        self.validators.append(validators.BRCPFValidator())
+
+
+class BRCNPJField(CharField):
+    """
+    A model field for the brazilian document named of CNPJ (Cadastro Nacional de Pessoa Jurídica)
```

Comment:
Please add `.. versionadded:: 2.1` to the new fields. Hopefully we can get this into the next release. :-)